config/environments/development.rb

```diff
@@ -86,6 +86,27 @@
     authentication:       'plain',
     enable_starttls_auto: true
   }
+  # prevent emails from going to anyone except the developer
+  class DeveloperMailInterceptor
+    # send all emails to the developer's git email address
+    def self.delivering_email(message)
+      email_target = nil
+      begin
+        `git config user.email`.strip
```

Comment:
We lost the assignment here:
```suggestion
        email_target = `git config user.email`.strip
```